src/AzureIoTHub.Portal/Shared/Models/v1.0/IoTEdgeModule.cs

```diff
@@ -23,6 +23,9 @@ public class IoTEdgeModule
         /// </summary>
         public string Version { get; set; }
```

Comment:
since ``Version`` is hard-coded, I guess there is no need to have it in the model

Reply:
I will remove it